src/search/search.component.ts

```diff
@@ -107,6 +107,10 @@ export class Search implements ControlValueAccessor {
 	 * Sets the value attribute on the `input` element.
 	 */
 	@Input() value = "";
+	/**
+	 * Sets the autocomplete attribute on the `input` element.
+	 */
+	@Input() autocomplete = "on";
```

Comment:
We should probably type hint the possible values

```suggestion
	@Input() autocomplete: "off" | "on" = "on";
```

Reply:
There are other options though. Why are we limiting the user to just on and off?

Reply:
I totally didn't realize there were more options ... maybe instead we should link out to somewhere that lists the options - like https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#Values

Reply:
you want me to include this as a comment?

Reply:
Yeah, something to that effect ... just so people are aware